Altis_Life.Altis/core/actions/fn_catchFish.sqf

```diff
@@ -2,29 +2,43 @@
 /*
     File: fn_catchFish.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified and optimised by SimZor
```

Comment:
We possibly need to agree on whether we keep just original author in file.

Adding names of editors can become quite lengthy if every editor adds their name after each change.